deploy/crds/ohmyglb.absa.oss_v1beta1_gslb_cr.yaml

```diff
@@ -29,3 +29,6 @@ spec:
             path: /
   strategy:
     type: roundRobin # Use a round robin load balancing strategy, when deciding which downstream clusters to route clients too
+    splitBrainThresholdSeconds: 300
+    dnsTtlSeconds: 30
```

Comment:
as it is used as reference CR definition, the documentation style comments will be useful there